pages/wishlist-page/wishlist.css

```diff
@@ -0,0 +1,51 @@
+body {
+    display: grid;
+    grid-template-rows: 4.5rem 1fr 3rem;
+    grid-template-areas: 
+    'nav'
+    'main'
+    'footer';
+    min-height: 100vh;
+}
+
+.main-nav {
+    grid-area: nav;
+}
+
+main {
+    grid-area: main;
+    
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    align-items: center;
+}
+
+
+/*wishlish section*/
+.wishlist-items {
+    max-width: 85%;
+
```

Comment:
remove spacing.

Reply:
Will take care of spacing. Thanks for your feedback.